scripts/exglobal_forecast.sh

```diff
@@ -395,6 +397,16 @@ $NLN $FIX_AM/${H2OFORC}                        $DATA/global_h2oprdlos.f77
 $NLN $FIX_AM/global_solarconstant_noaa_an.txt  $DATA/solarconstant_noaa_an.txt
 $NLN $FIX_AM/global_sfc_emissivity_idx.txt     $DATA/sfc_emissivity_idx.txt
 
+## merra2 aerosol climo
+for n in 01 02 03 04 05 06 07 08 09 10 11 12; do
+$NLN $FIX_AER/merra2.aerclim.2003-2014.m${n}.nc $DATA/aeroclim.m${n}.nc
+done
+cp -f $FIX_LUT/optics_BC.v1_3.dat $DATA/optics_BC.dat
+cp -f $FIX_LUT/optics_OC.v1_3.dat $DATA/optics_OC.dat
+cp -f $FIX_LUT/optics_DU.v15_3.dat $DATA/optics_DU.dat
+cp -f $FIX_LUT/optics_SS.v3_3.dat $DATA/optics_SS.dat
+cp -f $FIX_LUT/optics_SU.v1_3.dat $DATA/optics_SU.dat
+
```

Comment:
Please have your science reviewer check this. Also, why copies and not symlinks here? Thanks!

Reply:
@AnningCheng-NOAA @yangfanglin Can these copies become symlinks?